Our binutils somehow ended up with a -mfix-gs2f-kernel, rather than
-mfix-ls2f-kernel.  Cope with it.

Index: arch/mips/Makefile
===================================================================
--- arch/mips/Makefile.orig	2010-03-03 20:24:31.000000000 -0300
+++ arch/mips/Makefile	2010-03-03 20:24:36.000000000 -0300
@@ -137,6 +137,7 @@ cflags-$(CONFIG_CPU_LOONGSON2E) += \
 cflags-$(CONFIG_CPU_LOONGSON2F) += \
 	$(call cc-option,-march=loongson2f,-march=r4600) \
 	$(call as-option,-Wa$(comma)-mfix-ls2f-kernel,) \
+	$(call as-option,-Wa$(comma)-mfix-gs2f-kernel,) \
 	$(call as-option,-Wa$(comma)-mfix-loongson2f-nop,) \
 	$(call as-option,-Wa$(comma)-mfix-loongson2f-jump,)
 
